### PR TITLE
wip:style-changes

### PR DESCRIPTION
**Pull request by DavitKhachatryan110**

### CODE REVIEW

File: js/packages/common/src/components/CurrentUserBadge/index.tsx [hunk 1]
```diff
@@ -20,22 +20,23 @@ export const CurrentUserBadge = (props: {
...
-      };
+      display: 'flex',
+      width: props.iconSize || 20,
+      paddingLeft: 0,
+      borderRadius: 50,
+    };
 
   const baseWalletKey: React.CSSProperties = {
-    height: props.iconSize,
+    height: "33px",
```
Comment (Shitikyan) [inline js/packages/common/src/components/CurrentUserBadge/index.tsx:36]: Revert

File: js/packages/web/src/views/collections/index.less [hunk 2]
```diff
@@ -58,32 +78,37 @@
...
-  background:linear-gradient(90deg, #851CEF 3%, #24DDAF 100%) !important;
-}
-@media only screen and (max-width: 992px) {
-  .card-container{
-    min-height: 499px;
-  }
+  background: linear-gradient(90deg, #851CEF 3%, #24DDAF 100%) !important;
 }
+
+// @media only screen and (max-width: 992px) {
```
Comment (Shitikyan) [inline js/packages/web/src/views/collections/index.less:110]: remove

File: js/packages/web/src/views/latestsale/index.less [hunk 3]
```diff
@@ -1,7 +1,7 @@
 /* top sec starts */
-.ant-modal-body {
-  background-color: #1f2331;
-}
+// .ant-modal-body {
```
Comment (Shitikyan) [inline js/packages/web/src/views/latestsale/index.less:2]: remove

Comment (DavitKhachatryan110) [reply js/packages/web/src/views/latestsale/index.less:2]: I still need it in the future

**Pull request merged**